findbugs/src/test/java/edu/umd/cs/findbugs/DetectorsTest.java

```diff
@@ -223,24 +228,16 @@ private void setUpEngine(String... analyzeMe) throws IOException {
             }
         }
 
-        // FIXME : the tests use our annotations, so we need them in the classpath...
-        String classpath = System.getProperty("java.class.path");
-        if (classpath == null) {
-            String rootDirectory;
-            try {
-                rootDirectory = new File(getClass().getResource("/").toURI()).getCanonicalPath();
-            } catch (final URISyntaxException e) {
-                throw new RuntimeException(e);
-            }
-            project.addAuxClasspathEntry(rootDirectory);
-        } else {
-            String[] cpParts = classpath.split(File.pathSeparator);
-            for (String cpStr : cpParts) {
-                File file = new File(cpStr);
-                if (file.isDirectory()) {
-                    project.addAuxClasspathEntry(file.getCanonicalPath());
+        File annotationsLib = new File("../spotbugs-annotations/build/libs");
```

Comment:
I guess we can simply delete `if (file.isDirectory())`, because gradle added jar file path to CLASSPATH.

```java
System.out.println("CP:" + System.getProperty("java.class.path"));
->
    ~/spotbugs/findbugs/build/classes/test:
    ...
    ~/spotbugs/spotbugs-annotations/build/libs/spotbugs-annotations-3.1.0-SNAPSHOT.jar
    ...
```